### PR TITLE
6527 Fixed market list so it updates when switching between categories

### PR DESCRIPTION
**Pull request by michelleinez**
[ Clubhouse Story](https://app.clubhouse.io/augur/story/6527)

# Submitter checklist
- [ ] Test covering functionality added/fixed in

Check the linked story includes the following:
- [x] Steps to reproduce
- [ ] Screenshot (If applicable)

# Reviewer Checklist

- [x] Test/lint pass
- [ ] Post merge, story marked complete

### CODE REVIEW

**Comment by tinybike** [comment]
This looks good to me, although I'm wondering why hasLoadedCategory is an object to begin with...

File: src/modules/markets/components/markets-view.jsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ function loadMarkets(options) {
     // Expected behavior is to load a specific category if one is present
     // else, if we aren't searching (which is a local market data search)
     // then load markets (loads all markets)
-    if (category && !options.hasLoadedCategory[category]) {
+    if (category) {
       options.loadMarketsByCategory(category)
     } else if (!category && !search) {
```
Comment (justinbarry) [inline src/modules/markets/components/markets-view.jsx]: Do you still need the `!category`?

**Comment by bthaile** [comment]
Did this branch go away, can not find branch in order to do review.

File: src/modules/markets/components/markets-view.jsx (hunk 1 above)
Comment (michelleinez) [reply src/modules/markets/components/markets-view.jsx]: Oops, no we don't.  :)

**Comment by justinbarry** [comment]
This is a PR pulling from here https://github.com/michelleinez/augur.

**Pull request merged**